### PR TITLE
Merge branch 'master' of https://github.com/Lemon1903/DS2-Finals-Fitness-App

### PR DESCRIPTION
**Pull request by reciopaolo**

### CODE REVIEW

**Review by Lemon1903 — APPROVED** [review]
Great!

File: main.py [hunk 1]
```diff
@@ -52,6 +66,13 @@ def build_app(self, *_) -> MDScreenManager:
 
         return manager_screens
 
+    def switch_theme_style(self):
+        self.theme_cls.primary_palette = (
+            "Indigo" if self.theme_cls.primary_palette == "Indigo" else "Indigo"
```
Comment (sbstntmprl) [inline main.py:72]: This is just a test comment.

**Review by sbstntmprl — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (Lemon1903) [reply main.py:72]: Can you fix this changes? can you remove that line

Comment (reciopaolo) [reply main.py:72]: ok

**Review by sbstntmprl — CHANGES_REQUESTED** [review]
Change "Indigo" to "DeepPurple" on line 71 in main.py

**Pull request closed without merging**